File Sizes cleanup

ihsaan-ullah opened the pull request:
# @ mention of reviewers
...


# A brief description of the purpose of the changes contained in this PR.
...


# Issues this PR resolves
...


# Todos:
```python

# Get the maximum ID for Data
from datasets.models import Data
latest_id_data = Data.objects.latest('id').id
print("Data Last ID: ", latest_id_data)


# Get the maximum ID for Submission
from competitions.models import Submission
latest_id_submission = Submission.objects.latest('id').id
print("Submission Last ID: ", latest_id_submission)


# Get the maximum ID for Submission Detail
from competitions.models import SubmissionDetails
latest_id_submission_detail = SubmissionDetails.objects.latest('id').id
print("SubmissionDetail Last ID: ", latest_id_submission_detail)



# Run the conversion only for records with id <= latest_id
from datasets.models import Data
for data in Data.objects.filter(id__lte=<latest_id>):
    if data.file_size:
        data.file_size = data.file_size * 1024  # Convert from KiB to bytes
        data.save()


# Run the conversion only for records with id <= latest_id
from competitions.models import Submission
for sub in Submission.objects.filter(id__lte=<latest_id>):
    if sub.file_size:
        sub.file_size = sub.file_size * 1024  # Convert from KiB to bytes
        sub.save()


# Run the conversion only for records with id <= latest_id
from competitions.models import SubmissionDetails
for sub_det in SubmissionDetails.objects.filter(id__lte=<latest_id>):
    if sub_det.file_size:
        sub_det.file_size = sub_det.file_size * 1024  # Convert from KiB to bytes
        sub_det.save()

```



# A checklist for hand testing
- [ ] add checklist here


# Any relevant files for testing
[link]('#') to any relevant files (or drag and drop into github)


# Checklist
- [ ] Code review by me 
- [ ] Hand tested by me 
- [ ] I'm proud of my work
- [ ] Code review by reviewer
- [ ] Hand tested by reviewer
- [ ] CircleCi tests are passing
- [ ] Ready to merge

Didayolo commented:
@ihsaan-ullah 

For the manual intervention, I replaced:

- `<latest_data_id>` by `latest_id_data`
- `<latest_sub_id>` by `latest_id_submission`
- `<latest_sub_det_id>` by `latest_id_submission_detail`

Is that correct?

ihsaan-ullah commented:
> For the manual intervention, I replaced:
> 
> * `<latest_data_id>` by `latest_id_data`
> * `<latest_sub_id>` by `latest_id_submission`
> * `<latest_sub_det_id>` by `latest_id_submission_detail`
> 
> Is that correct?

Yes, I have updated the code in the main comment with the right var name

ihsaan-ullah commented:
@Didayolo I have 
- fixed the manual intervention code
- added the missing migration
- resolved merge conflicts.

For this PR deployment we have to stop users from using the platform. I think @ObadaS  knows how to do it

Didayolo commented:
@ihsaan-ullah 

Great, thanks.

Not sure if it is related to this PR (or the other?), but the quota is written "B" instead of "GB":

<img width="222" alt="Capture d’écran 2025-03-24 à 22 34 17" src="https://github.com/user-attachments/assets/bbbaf148-bf37-4701-8a1a-c6cebe359627" />
